packages/backend/controllers/teacherController.js

```diff
@@ -92,7 +94,52 @@ const loginTeacher = async (req, res) => {
   });
 };
 
+const sendDirectMessageToStudent = async (req, res) => {
+  const { messageHeader, messageBody, studentID } = req.body;
+
+  const teacherID = res.locals.user;
```

Comment:
res.locals.user is the full teacher object.
Teacher_ID = res.locals.user._id
Classroom_ID = res.locals.user.classroom